valetudo-map-card.js

```diff
@@ -614,20 +710,15 @@ class ValetudoMapCard extends HTMLElement {
     if (this._config.crop.right === undefined) this._config.crop.right = 0;
     if (this._config.min_height === undefined) this._config.min_height = 0;
```

Comment:
I like the new DEFAULT_CARD_CONFIG thing but why are these few lines not there when the rest is?

Reply:
That's because `Object.assign()` isn't nested, meaning that it wouldn't correctly apply the contents of config.crop

```
    // Crop settings
    if (this._config.crop !== Object(this._config.crop)) this._config.crop = {};
    if (this._config.crop.top === undefined) this._config.crop.top = 0;
    if (this._config.crop.bottom === undefined) this._config.crop.bottom = 0;
    if (this._config.crop.left === undefined) this._config.crop.left = 0;
    if (this._config.crop.right === undefined) this._config.crop.right = 0;
    if (this._config.min_height === undefined) this._config.min_height = 0;
```

config.min_height is an oversight though 🤔

The config.rotate assigment also has special needs if I understand it correctly

Reply:
Well let's move min_height then and sure the rest is okay.

All the code looks fine, just going to run it on my device for a day before I merge as a "smoke test"